src/components/home/HomeInfo.jsx

```diff
@@ -0,0 +1,137 @@
+import styled from 'styled-components';
+import { icHand, imgChuncheon, imgSokcho, imgDaejeon } from 'assets';
+
+function HomeInfo() {
+  return (
+    <>
+      <h1>예정된 예약</h1>
+      <StyledFirstSection>
+        <StyledIcHand src={icHand} />
+        <span>아직 예약된 여행이 없습니다!</span>
+        <StyledSearchButton>숙소 검색하기</StyledSearchButton>
+      </StyledFirstSection>
+      <h2>이전 여행지</h2>
+      <StyledSecondSection>
+        <StyledImg>
+          <img src={imgChuncheon} />
+          <img src={imgSokcho} />
+          <img src={imgDaejeon} />
+        </StyledImg>
+        <StyledInfo>
+          <div>
+            <p>Geunhwa-dong,Chuncheon</p>
+            <span>2022년 1월 12일 - 2022년 1월 14일</span>
+          </div>
+          <div>
+            <p>joyang-dong,Sokcho</p>
+            <span>2021년 12월 5일 - 2021년 12월 6일</span>
+          </div>
+          <div>
+            <p>jayang-dong,Daejeon</p>
+            <span>2021년 6월 17일 - 2021년 7월 1일</span>
```

Comment:
정말 사소한 내용이지만,, 피그마 확인해 보니까 띄어쓰기 있는 게 좋을 것 같아서 남겨!
```suggestion
            <p>Geunhwa-dong, Chuncheon</p>
            <span>2022년 1월 12일 - 2022년 1월 14일</span>
          </div>
          <div>
            <p>joyang-dong, Sokcho</p>
            <span>2021년 12월 5일 - 2021년 12월 6일</span>
          </div>
          <div>
            <p>jayang-dong, Daejeon</p>
            <span>2021년 6월 17일 - 2021년 7월 1일</span>
```